src/powershell/install-chocolatey.ps1

```diff
@@ -0,0 +1,32 @@
+# Download and execute Chocolatey install script
+Set-ExecutionPolicy Bypass -Scope Process -Force
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072
+iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+
+# Configure Chocolatey to use FireEye's package repository
+iex "choco sources add --name fireeye --source https://www.myget.org/F/fireeye/api/v2 --priority 1"
+iex "choco feature enable --name allowGlobalConfirmation"
+iex "choco feature enable --name allowEmptyChecksums"
+Write-Output "[*] Chocolatey successfully configured to use FireEye's package repository"
+
+# Disable timeout settings to help with packages that take a long time to install
+& powercfg -change -monitor-timeout-ac 0 | Out-Null
+& powercfg -change -monitor-timeout-dc 0 | Out-Null
+& powercfg -change -disk-timeout-ac 0 | Out-Null
+& powercfg -change -disk-timeout-dc 0 | Out-Null
+& powercfg -change -standby-timeout-ac 0 | Out-Null
+& powercfg -change -standby-timeout-dc 0 | Out-Null
+& powercfg -change -hibernate-timeout-ac 0 | Out-Null
+& powercfg -change -hibernate-timeout-dc 0 | Out-Null
```

Comment:
If the order doesn't matter here, we might as well alphabetize:
```suggestion
& powercfg -change -disk-timeout-ac 0 | Out-Null
& powercfg -change -disk-timeout-dc 0 | Out-Null
& powercfg -change -hibernate-timeout-ac 0 | Out-Null
& powercfg -change -hibernate-timeout-dc 0 | Out-Null
& powercfg -change -monitor-timeout-ac 0 | Out-Null
& powercfg -change -monitor-timeout-dc 0 | Out-Null
& powercfg -change -standby-timeout-ac 0 | Out-Null
& powercfg -change -standby-timeout-dc 0 | Out-Null
```

If the order does matter here, feel free to ignore this suggestion.

Reply:
@dav3r resolved with [199e5a1](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/199e5a1c77e26deaeb021bda8b6c2ff8f020f5b5)